package.json

```diff
@@ -17,12 +17,12 @@
     "build": "babel src  --out-dir production",
     "start": "node production/server.js",
     "migrate:drop": "sequelize db:migrate:undo:all",
+    "seed:create": "npx sequelize-cli db:seed:all",
```

Comment:
Remove the seed command as you are having no seeds

Reply:
I have rebase after Gad's merge. And we discussed and told me that he will need it. Once i remove them, his test that i have are failing

Reply:
Then you shouldn't modify the script. Develop already has a script to run seeders.

Reply:
Sorry for modifying.Will not happen again